src/ua/artcode/udiary/utils/Validator.java

```diff
@@ -1,45 +1,133 @@
 package ua.artcode.udiary.utils;
 
 import ua.artcode.udiary.exception.AppException;
-import ua.artcode.udiary.model.Dairy;
-import ua.artcode.udiary.model.Record;
-import ua.artcode.udiary.model.User;
+import ua.artcode.udiary.model.*;
+
+import java.util.List;
 
 public class Validator {
 
+    /**
+     * This method validates new record before storing into DB.
+     * @param record new record, which must not be null,
+     *               must have not null and not empty title,
+     *               not null and not empty body.
+     * @throws AppException when record is null, its title is null or empty,
+     * or its body is null or empty.
+     *
+     * @see Record
+     * @see AppException
+     * */
     public static void validateRecord(Record record) throws AppException {
```

Comment:
Validator should throw ValidationException that must be inherited from AppException

Reply:
Created and fixed. Please, check new version in the next push.